src/get_select_cfs.R

```diff
@@ -0,0 +1,88 @@
+#' Divergent climate models were selected for the park and koppen centroids. 
+#' This function selects which to use based on either the shortest distance
+#' between the centroid and watersupply watershed (method = "distance") or 
+#' the koppen centroid that matches the watersupply watershed (method = "koppen") 
+#' 
+#' @param park_boundary
+#' @param watersupply_watershed
+#' @param method
+#' 
+#' @returns select_cfs dataframe containing the selected climate futures for
+#' the selected centroid.
+#' 
+get_select_cfs <- function(park_boundary, watersupply_watershed, method) {
+  
+  # Get koppen climate grid for the park
+  koppen_park <- get_koppen(aoi = park_boundary, 
```

Comment:
I don't see this used downstream, is it needed?

Reply:
A casualty of the merge conflict. Added back into CCVA_Processing line 271